apps/meteor/client/hooks/useAppActionButtons.ts

```diff
@@ -42,6 +41,13 @@ export const useAppActionButtons = <TContext extends `${UIActionButtonContext}`>
 		[],
 	);
 
+	const { status } = useConnectionStatus();
```

Comment:
Please use `status` in the query key and disable the query if `status !== 'connected;` as we discussed in dm